torch/testing/_internal/common_utils.py

```diff
@@ -1581,7 +1581,11 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
         torch._inductor.config.fallback_random = True
 
 
-def xpassIfTorchDynamo(func):
+# seems like this is only used in test/torch_np
+def xpassIfTorchDynamo_np(func):
+    # numpy 2.0+ is causing issues
```

Comment:
This is exposed only with Python >= 3.13? Should we create an issue to investigate later?